### ISSUE TITLE
Colin McRae 2005 Coredump crash with Vulkan only

### ISSUE THREAD

**Issue by poVoq** [hrydgard/ppsspp#13545]
# What happens?
Colin McRae 2005 works mostly fine with OpenGL renderer on Linux PPSSPP version 1.10.x (some graphical glitches though) with a Intel GPU.
Switching to Vulkan though results in a white screen and a "core dump" crash after trying to enter a race; menu still works fine.
Other PSP games work fine with Vulkan renderer.

### What *should* happen?
Should work like on OpenGL.

**Comment by hrydgard** [comment]
I have a similar issue in Burnout Legends where the car selection screen crashes on Intel only. Suspect a driver bug but hard to prove...  It passes validation layers.

**Comment by unknownbrackets** [comment]
Has this gotten any better with updated drivers?

-[Unknown]

**Comment by poVoq** [comment]
No, still the same with the latest mesa drivers for intel.

**Comment by hrydgard** [comment]
Yeah, still happening in Burnout as well on mesa drivers. I tried narrowing it down a while ago, it's got something to do with color image copies but I failed to figure out the exact call causing it. Maybe there isn't one and it's just some timing issue in the driver...
